### PR TITLE
Change Subscripts in GamePhys

### PR DESCRIPTION
**Pull request by Ant13731**
Changed all subscript `I` into `j` for GamePhys.
Closes issue #2370

### CODE REVIEW

File: code/drasil-example/Drasil/GamePhysics/DataDefs.hs [hunk 1]
```diff
@@ -43,7 +43,7 @@ ctrOfMass = mkQuantDef posCM ctrOfMassEqn
 
 -- FIXME (Variable "i") is a horrible hack
 ctrOfMassEqn :: Expr
-ctrOfMassEqn = sumAll (Variable "i") (sy massI * sy posI) / sy mTot
+ctrOfMassEqn = sumAll (Variable "i") (sy massj * sy posj) / sy mTot
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GamePhysics/DataDefs.hs:46]: Hmm, something here seems amiss. The hack here is likely quite wrong. It does look like `mass` and `pos` depend on `j` (as defined in `Unitals`), but this sum over `i` is then wrong. Could you check in the generated PDF and HTML if that's indeed the case?

File: code/drasil-example/Drasil/GamePhysics/DataDefs.hs [hunk 2]
```diff
@@ -319,7 +319,7 @@ momentOfInertia :: QDefinition
 momentOfInertia = mkQuantDef QP.momentOfInertia momentOfInertiaEqn
 
 momentOfInertiaEqn :: Expr
-momentOfInertiaEqn = sumAll (Variable "i") $ sy massI * (sy rRot $^ 2)
+momentOfInertiaEqn = sumAll (Variable "i") $ sy massj * (sy rRot $^ 2)
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GamePhysics/DataDefs.hs:322]: Here as well, of course.

File: code/drasil-example/Drasil/GamePhysics/Unitals.hs [hunk 3]
```diff
@@ -171,23 +171,23 @@ posCM = ucs "p_CM" (nounPhraseSP "Center of Mass")
...
                 (cn "of the j-th particle")) (phrase QPP.mass)) 
                 (sub (eqSymb QPP.mass) lJ) Real kilogram
 
-posI = ucs' (dccWDS "p_j" (compoundPhrase' (QP.position ^. term) 
+posj = ucs' (dccWDS "p_j" (compoundPhrase' (QP.position ^. term) 
                (cn "vector of the j-th particle")) (phrase QP.position))
                (sub (eqSymb QP.position) lJ) Real metre
 
-accI = ucs' (dccWDS "accI" (compoundPhrase' (cn "the i-th body's")
+accj = ucs' (dccWDS "accj" (compoundPhrase' (cn "the i-th body's")
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GamePhysics/Unitals.hs:182]: Here `accj` has `i` in its English form!

File: code/drasil-example/Drasil/GamePhysics/Unitals.hs [hunk 4]
```diff
@@ -171,23 +171,23 @@ posCM = ucs "p_CM" (nounPhraseSP "Center of Mass")
...
   (sub (eqSymb QP.position) lCMass) Real metre
 
-massI = ucs' (dccWDS "m_j" (compoundPhrase' (QPP.mass ^. term)
+massj = ucs' (dccWDS "m_j" (compoundPhrase' (QPP.mass ^. term)
                 (cn "of the j-th particle")) (phrase QPP.mass)) 
                 (sub (eqSymb QPP.mass) lJ) Real kilogram
 
-posI = ucs' (dccWDS "p_j" (compoundPhrase' (QP.position ^. term) 
+posj = ucs' (dccWDS "p_j" (compoundPhrase' (QP.position ^. term) 
                (cn "vector of the j-th particle")) (phrase QP.position))
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GamePhysics/Unitals.hs:179]: and here `pos` has `j` in its English form - this is inconsistent.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

File: code/drasil-example/Drasil/GamePhysics/Unitals.hs (hunk 3 above)
Comment (Ant13731) [reply code/drasil-example/Drasil/GamePhysics/Unitals.hs:182]: Should I change the English description to "j-th body" in order to match the other variable changes. And does the same go for "F_i"?

Comment (Ant13731) [reply code/drasil-example/Drasil/GamePhysics/Unitals.hs:182]: I also noticed that torque and velocity use the "i-th" body in their description too. Should I change those to "j-th"?

Comment (JacquesCarette) [reply code/drasil-example/Drasil/GamePhysics/Unitals.hs:182]: They should all consistently use the same variable, for now. Putting a variable name in there is a hack, but one we have to live with right now.  Let's do `j`, as `i` is over-used and could clash with other things.

**Review by JacquesCarette — APPROVED** [review]

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**